<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>241</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QCheckBox" name="checkBox_isWAN">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>180</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>公网登录</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_secretKey">
      <property name="geometry">
       <rect>
        <x>78</x>
        <y>134</y>
        <width>50</width>
        <height>11</height>
       </rect>
      </property>
      <property name="text">
       <string>私钥：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_login">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>179</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ip">
      <property name="geometry">
       <rect>
        <x>148</x>
        <y>39</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_port">
      <property name="geometry">
       <rect>
        <x>148</x>
        <y>69</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_secretKey">
      <property name="geometry">
       <rect>
        <x>148</x>
        <y>129</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_ip">
      <property name="geometry">
       <rect>
        <x>78</x>
        <y>44</y>
        <width>50</width>
        <height>11</height>
       </rect>
      </property>
      <property name="text">
       <string>IP地址：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_publicKey">
      <property name="geometry">
       <rect>
        <x>148</x>
        <y>99</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_publicKey">
      <property name="geometry">
       <rect>
        <x>78</x>
        <y>104</y>
        <width>50</width>
        <height>11</height>
       </rect>
      </property>
      <property name="text">
       <string>公钥：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_port">
      <property name="geometry">
       <rect>
        <x>78</x>
        <y>74</y>
        <width>50</width>
        <height>11</height>
       </rect>
      </property>
      <property name="text">
       <string>端口：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_getKey">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>210</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <underline>true</underline>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color:blue;
border:none;</string>
      </property>
      <property name="text">
       <string>获取密钥</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">text-align:center;</string>
      </property>
      <property name="text">
       <string>请添加微信联系</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>165</width>
        <height>165</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/QR.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="button_returnLogin">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>210</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <underline>true</underline>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color:blue;
border:none;</string>
      </property>
      <property name="text">
       <string>返回登录</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_ip</tabstop>
  <tabstop>lineEdit_port</tabstop>
  <tabstop>lineEdit_publicKey</tabstop>
  <tabstop>lineEdit_secretKey</tabstop>
  <tabstop>checkBox_isWAN</tabstop>
  <tabstop>button_login</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
